pkgs/applications/misc/mepo/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromSourcehut, pkg-config, zig
+, curl, SDL2, SDL2_image, SDL2_ttf
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mepo";
+  version = "0.2";
+
+  src = fetchFromSourcehut {
+    owner = "~mil";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ECq748GpjOjvchzAWlGA7H7HBvKNxY9d43+PTOWopiM=";
+  };
+
+  nativeBuildInputs = [ pkg-config zig ];
+
+  buildInputs = [ curl SDL2 SDL2_image SDL2_ttf ];
+
+  buildPhase = ''
+    export HOME=$TMPDIR
+    zig build -Drelease-safe=true -Dcpu=baseline
```

Same goes for all the other phases please

```suggestion
    runHook preBuild
    
    export HOME=$TMPDIR
    zig build -Drelease-safe=true -Dcpu=baseline
    
    runHook postBuild
```

---

I'm curious, what's the purpose of this?

---

So that overlays for the package run all phases for the package as expected